crates/egui/src/widgets/drag_value.rs

```diff
@@ -475,13 +475,15 @@ impl<'a> Widget for DragValue<'a> {
                     .desired_width(ui.spacing().interact_size.x)
                     .font(text_style),
             );
-            let parsed_value = match custom_parser {
-                Some(parser) => parser(&value_text),
-                None => value_text.parse().ok(),
-            };
-            if let Some(parsed_value) = parsed_value {
-                let parsed_value = clamp_to_range(parsed_value, clamp_range.clone());
-                set(&mut get_set_value, parsed_value);
+            if response.lost_focus() {
```

Comment:
It would be nice if you also added a comment justifying why this check should be done. The fact that it was absent before allows us to conclude that it is not self evident. Description of the reasons why this check was added and what it affects will make it easier to resolve problems if this change leads to them.

A good tone will also leave a link to the GitHub Issue, where you can get more information.